apps/stress-test/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@fluentui/stress-test",
```

Comment:
Are we publishing this app ? perhaps we should keep it private until we have full feature coverage (scoped tests, decoupling from out libraries) before we want to publish this

Reply:
Done!